VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/video-call-models.ts

```diff
@@ -15,12 +15,12 @@ export class CallError {
 }
 
 export class ParticipantUpdated {
-    public isMuted: boolean;
+    public isRemoteMuted: boolean;
     public handRaised: boolean;
     public pexipDisplayName: string;
 
     constructor(isMuted: string, buzzTime: number, pexipName: string) {
-        this.isMuted = isMuted === 'YES';
+        this.isRemoteMuted = isMuted.toUpperCase() === 'YES';
```

Comment:
by is this a string? why not just make it a boolean from the get go

Reply:
pexip post updates as a string unfortunately. as seen [here](https://docs.pexip.com/api_client/api_pexrtc.htm#onRosterList)